app/models/plugins/has_form.rb

```diff
@@ -33,7 +33,7 @@ def dup
     clone = super
     clone.save!
 
-    clone.form.form_elements = form.form_elements.map(&:dup) if clone.form
+    clone.form&.form_elements = form.form_elements.map(&:dup)
```

Comment:
That's interesting! I didn't know you could use the safe operator on the left side of an assignment. This means that it's clearly doing more than the old ActiveSupport `try`. 

However, my comment still stands. I was refering to the right side of the assignment where you do `form.form_elements`. If form is nil this will blow up.

```
>> house = Object.new
=> #<Object:0x007ff8c092ce38>
>> house.windows
NoMethodError: undefined method `windows' for #<Object:0x007ff8c092ce38>
	from (irb):3
	from /Users/rod/.rvm/rubies/ruby-2.3.3/bin/irb:11:in `<main>'
>> house&.windows = house.windows
NoMethodError: undefined method `windows' for #<Object:0x007ff8c092ce38>
	from (irb):4
	from /Users/rod/.rvm/rubies/ruby-2.3.3/bin/irb:11:in `<main>'
```